docs/_specification/1.2-DRAFT/root-data-entity.md

```diff
@@ -128,14 +127,15 @@ be minimally valid.
 
 ## Direct properties of the Root Data Entity
 
-The _Root Data Entity_ MUST have the following properties:
+The _Root Data Entity_ of a _Valid RO-Crate Dataset_  MUST have the following properties:
```

Comment:
terminology _Valid RO-Crate Dataset_ is new, what does this mean?

Reply:
It was left over from the other change - sorry I will remove all that